### DISCUSSION TITLE
初次使用Toga遇到问题，Encountering problems when using Toga for the first time。

### DISCUSSION THREAD

**Discussion by RONG-GUI** [beeware/toga#2748] [Q&A] [answered]
1、使用官方示例，我不需要菜单栏，所以把MainWindow换成Window后，在运行时报错了【1、Using the official example, I didn't need a menu bar, so when I changed MainWindow to Window, I reported an error during runtime】：“”“ValueError: Application does not have a main window. Does your startup() method assign a value to self.main_window?“”“

2、翻阅了问题讨论，同样的：最小化窗口后，不能复原原来的窗口大小？【2. After reviewing the discussion of the issue, it was found that some users have suggested that after minimizing the window, the original window size cannot be restored?】

所以上述问题有解决方法不？谢谢【So is there a solution to the above problem? thank you】
ps：官方示例地址【Official Example Address】：https://toga.readthedocs.io/en/latest/reference/api/window.html

**Comment by mhsmith** [comment, marked as answer]
> Official Example Address】：https://toga.readthedocs.io/en/latest/reference/api/window.html

The "latest" version of the documentation contains things we haven't released yet (#2708). Please use the "stable" version instead. You can switch between them with the green menu at the bottom left of the page, or by editing the URL.

> I didn't need a menu bar, so when I changed MainWindow to Window, I reported an error during runtime

The option to use a Window as the `main_window` will be added in the next version of Toga. In the stable version of Toga you need to use a MainWindow.

> after minimizing the window, the original window size cannot be restored

We're aware of this issue (#2729), but we don't have a solution for it at the moment.
